src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const stateObject = state;
+
+  for (const key of actions) {
+    if (key.type === 'addProperties') {
```

Comment:
This is clean code but as per checklist
2. [CODE STYLE]: Use switch statement if you have limited amount of conditions.

Reply:
changed to switch case.